spec/lib/nationbuilder/client_spec.rb

```diff
@@ -50,6 +50,7 @@
     it 'should handle a parametered GET' do
       VCR.use_cassette('parametered_get') do
         response = client.call(:basic_pages, :index, site_slug: 'organizeralexandreschmitt', limit: 11)
+        expect(response['status_code']).to eq(200)
```

Comment:
It looks like that would break the current `delete` interface, as it returns `true` or `false`, not a hash.  I'm going to leave it unless you think we can change the `delete` interface without anyone complaining